DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/EmailDelegates/EmailDelegatesViewModel.cs

```diff
@@ -46,6 +46,7 @@ EmailDelegatesFormData formData
         public int? Day { get; set; }
         public int? Month { get; set; }
         public int? Year { get; set; }
+        private readonly IClockUtility clockUtility = new ClockUtility();

```

Comment:
It would be better to pass this in in the constructor - c.f. [dependency inversion](https://betterprogramming.pub/straightforward-simple-dependency-inversion-vs-dependency-injection-7d8c0d0ed28e)

Reply:
Or even just pass the current time in, would be even better.